lambda/docker-runner.yaml

```diff
@@ -0,0 +1,111 @@
+# This template depends on having an AWS EFS in place which can be deployed with
+# https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EFS/efs.yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: S3Objects
+Description: >-
+  Docker runner product
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Container Image
+        Parameters:
+          - ImageUri
+          - EntryPoint
+          - Command
+          - WorkingDirectory
+          - MemorySize
+      - Label:
+          default: Synapse
+        Parameters:
+          - SynapseUser
+          - SynapsePassword
+      - Label:
+          default: Execution Trigger
+        Parameters:
+          - EnableSchedule
+          - Schedule
+Parameters:
+  MemorySize:
+    Type: Number
+    MinValue: 128
+    MaxValue: 10240
+    Default: 128
+    Description: >
+      The amount of memory available to the function at runtime. Increasing the function's
+      memory also increases its CPU allocation.
+  ImageUri:
+    Type: String
+    Description: >
+      The URI of the Amazon ECR) repository for the Lambda function's container image.
+      Example: 111111111.dkr.ecr.us-east-1.amazonaws.com/MY-DOCKER:latest
+  EntryPoint:
+    Type: String
```

Comment:
I'm pretty sure (but not 100% sure) that EntryPoint should be a list, like CMD is, https://phoenixnap.com/kb/docker-run-override-entrypoint

Reply:
changed to a list